global.css

```
@@ -0,0 +1,68 @@
+    --color-completed: #c8e6c9;
+    --color-badge-indica: #e8f5e9;
+    --color-badge-sativa: #fffde7;
+    --color-badge-hybrid: #f3e5f5;
+
+    /* Dark mode colors */
+    --color-dark-bg: #0a1410;
+    --color-dark-bg-elevated: #1a2820;
+    --color-dark-bg-card: #1e2d24;
+    --color-dark-border: #2a3d32;
+    --color-dark-border-bright: #3d5a47;
+    --color-primary-bright: #00ff88;
```

Stylelint (17.2.0)

[error] Expected "#00ff88" to be "#0f8" (color-hex-length)

(color-hex-length)